src/components/ButtonGroupPicker/index.js

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import StyledContainer from './styled/container';
+import StyledButtonGroup from './styled/buttonGroup';
+import StyledLegend from './styled/label';
+import { uniqueId } from '../../libs/utils';
+import { Provider } from './context';
+import RenderIf from '../RenderIf';
+import HelpText from './styled/helpText';
+import ErrorText from './styled/errorText';
+
+/**
+ * ButtonGroupPicker can be used to group related options. The ButtonGroupPicker will control the selected state of its child ButtonOption.
+ */
+class ButtonGroupPicker extends Component {
+    constructor(props) {
+        super(props);
+        this.groupNameId = this.props.name || uniqueId('options');
+        this.errorMessageId = uniqueId('error-message');
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    getValue() {
+        const { value } = this.props;
+        if (typeof value === 'string') {
+            return [];
+        }
+        return value;
+    }
+
+    getErrorMessageId() {
+        const { error } = this.props;
+        if (error) {
+            return this.errorMessageId;
+        }
+        return undefined;
+    }
+
+    handleOnChange(event) {
+        const { value, checked } = event.target;
+        const { value: values, multiple, onChange } = this.props;
```

Comment:
for better readability we can use value normally and value from event as eventValue:
```
const { value: eventValue, checked } = event.target;
const { value, multiple, onChange } = this.props;
```